frontend/src/screens/learner/card.tsx

```diff
@@ -126,150 +118,325 @@ const CompleteFlag: React.FC<StudyCardProps> = ({ study }) => {
+                    return acc + feedbackType + '.'
+                }, '')}`} />
+
+
+            <StudyDetailsItem icon='person' title='Principal Investigator' 
+                desc={ pi?.firstName + ' ' + pi?.lastName + ' | Institution: ' + pi?.institution } />
+
+            <StudyDetailsItem icon='person' title='Study Lead' 
+                desc={lead?.firstName + ' ' + lead?.lastName + ' | Institution: ' + lead?.institution } />
+
+            <Button
+                color='purple'
+                loading={isBusy}
+                data-analytics-select-content
+                data-content-type='study-launch'
+                data-content-tags={`,learning-path=${study.learningPath?.label},is-new-user=${env.isNewUser},`}
+                disabled={!isStudyLaunchable(study)}
+                data-testid='launch-study'
+                onClick={onLaunch}
+            >
+                {action} study
+            </Button>
+        </Flex>
     )
 }
 
-const PointsAndDuration: FC<StudyCardProps> = ({ study }) => {
-    const isMobile = useIsMobileDevice();
-
+const StudyDetailsItem: React.FC<{icon: IconSpec, title: string, desc: string | undefined}> = ({ icon, title, desc = '' }) => {
     return (
-        <Box className={cx({ 'small': !isMobile, 'xx-small': isMobile }, 'mt-auto', 'pt-1')} justify='between' align='center' wrap>
-            <Box gap='small'>
-                <Tag tag={study.subject} />
-            </Box>
-            <Box gap='small'>
-                {isMultiSession(study) && <span>*Total</span>}
-                <span>{getStudyDuration(study)} min</span>
-                &middot;
-                <span>{getStudyPoints(study)} pts</span>
-            </Box>
-        </Box>
+        <Flex align='flex-start' gap='sm' direction='column'>
+            <Flex align='center' gap='sm'>
+                <Icon icon={icon} color={colors.purple} />
+                <Text fw='bold' size='sm'>{title}</Text>
+            </Flex>
+            <Text size='sm'>
+                {desc}
+            </Text>
+        </Flex>
     )
 }
 
-export const StudyCard: React.FC<StudyCardProps> = ({ study }) => {
+export const StudyCard: React.FC<{study: ParticipantStudy, index: Number, hovered: Number, setHovered: Function}> = ({ study, index, hovered, setHovered }) => {
     const nav = useNavigate()
-    const onClick = () => nav(`/studies/details/${study.id}`)
+    const api = useApi()
+    const isMobile = useIsMobileDevice()
+    const isTablet = useIsTabletDevice()
+
+    const [timer, setTimer] = useState<number | null>(null)
+    const [element, setElement] = useState<HTMLDivElement | null>(null)
+    const [multiSessionShadow, setMultiSessionShadow] = useState<boolean>(false)
+    const [position, setPosition] = useState<{top: Number, left: Number}>({ top: 0, left: 0 })
+
+    useEffect(() => {
+
+        const buffer = { top: isTablet? -150: -150, left0: isTablet? 255: 18, leftRest: isTablet? -560: -790 }
+        if (element){
```

Comment:
Since the card moves as it is scrolled, I added the resize event listener, I also added the resize window listenner, in case the user changes the size of the window. The <StudyDetails /> component is positioned absolutely, so is moved based on the parent of the horizontally scrolled container, because on setting overflowX to scroll, no matter the value of overflowY, it being defaulted to auto, so in order to get around this issue, I positioned it based on the parent of the scrolling container.